servicecontrol/migrations/migrate-data-between-instances.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Migrate ServiceControl data between instances
+summary: Instructions on how to migrate ServiceControl data between instances.
+reviewed: 2025-06-12
+component: ServiceControl
+related:
+    - servicecontrol/migrations/replacing-audit-instances
+    - servicecontrol/migrations/replacing-error-instances
+    - servicecontrol/backup-sc-database
+redirects:
+    - servicecontrol/data-migration
+---
+
+ServiceControl, which exists to serve the management of distributed systems, is itself a distributed system. As a result, migrating all ServiceControl data between deployments means each piece of the system that accesses and writes data is migrated separately.
+
+This document describes in general terms how to migrate ServiceControl data, and links to more specific information on how to accomplish data migration for each piece of the system.
+
+## Overview
+
+ServiceControl data consists of audit and error message data, managed by [audit instances](/servicecontrol/audit-instances/) and [error instances](/servicecontrol/servicecontrol-instances/) respectively. To migrate all ServiceControl data from one deployment to another means replacing both audit and error instances.
```

Comment:
Could this be confusing because it is not clear what deployment to deployment means? Reading this someone might wonder whether that procedure is necessary for each patch or minor deployment or whether this is only necessary when doing major to major...

Or it might be necessary to indicate when switching from one instance to another is required, desirable or even encouraged to give some more hints